src/mbgl/renderer/render_layer.cpp

```diff
@@ -36,7 +36,7 @@ bool RenderLayer::hasRenderPass(RenderPass pass) const {
 }
 
 bool RenderLayer::needsRendering() const {
-    return passes != RenderPass::None
```

Comment:
Now, RenderLayer::passes and evaluatedProperties->renderPasses is the same (as set i evaluate()). Why the change?

Reply:
leftover, thanks for the catch!